Practice Areas/Engineering/RFC/001-internal-tools-access.md

```diff
@@ -0,0 +1,56 @@
+# Internal Tool Access
+
+The following tools are currently being used by developers and operators for the VSP, currently exposed by the SOCKS proxy. This proxy adds a good deal of toil for operations staff, is not scaleable for onboarding non-VSA engineers to the platform, and is unnecessary now that we have access to VAEC.
+
+Below is a chart with tools we maintain today along with perceived risk profile and exposure tolerance. Risk profile is meant to describe the amount of risk to Veterans and VA if information or access were compromised. Exposure tolerance is meant to describe whether VA might tolerate a service being exposed internally or externally from a compliance perspective.
+
+| Tool | Risk Profile | Exposure Tolerance |
```

Comment:
Is authentication for these tools a separate discussion?  Sentry still has accounts for people who no longer work on VA projects, but it didn't matter because they couldn't reach the url without socks and account creation is wide open on Grafana, I think.

Reply:
Yeah, we're working on GitHub OAuth that will ensure a user is part of the VA org on GitHub. Grafana is now setup that way, we ran into some issues with Sentry due to what appears to be a bug in pyOpenSSL. We may get some pressure to attempt to get these setup with AD authentication, but I don't know if that's possible or what it looks like yet.